examples/acquiring_data/downloading_and_plotting.py

```diff
@@ -0,0 +1,37 @@
+"""
+=============================
+Downloading and plotting data
+=============================
+"""
+###############################################################################
+# Import the required modules.
+from sunpy.net import Fido, attrs
+import sunpy.map
+import matplotlib.pyplot as plt
+
+###############################################################################
+# Define the timerange and instrument.
+timerange = attrs.Time('1996/05/24 11:00', '1996/05/24 12:00')
+instrument = attrs.Instrument('C3')
+
+###############################################################################
+# Search for data that is from the requested instrument and in the requested
+# timerange.
+result = Fido.search(timerange, instrument)
+result
```

Comment:
If we use this as a script `results` won't be printed. If you want them to be printed, then we should have `print(results)`.